models/evc.py

```diff
@@ -137,7 +138,8 @@ def __init__(self, controller, **kwargs):
 
     def sync(self):
         """Sync this EVC in the storehouse."""
-        self._storehouse.save_evc(self)
+        #self._storehouse.save_evc(self)
+        self._mongo_controller.upsert_evc(self.as_dict())
         log.info(f"EVC {self.id} was synced to the storehouse.")
```

Comment:
Should we take the opportunity to remove this `log.info(f"EVC {self.id} was synced to the storehouse.")`? I'm not if it's relevant for operators, the other ones like `was deployed` seemed more relevant for them.